### PR TITLE
do not bundle when not in a gem and fail when bundle fails

### PR DESCRIPTION
**Pull request by tamgrosser**
@gregorym 
/fyi @grosser :heart:

### CODE REVIEW

File: lib/bump.rb [hunk 1]
```diff
@@ -59,9 +59,9 @@ def current
 
       def bump(file, current, next_version, options)
         replace(file, current, next_version)
-        if options[:bundle] and under_version_control?("Gemfile.lock")
+        if options[:bundle] and Dir.glob('*.gemspec').any? and under_version_control?("Gemfile.lock")
           bundler_with_clean_env do
-            system("bundle")
+            return ["Bundle error", 1] unless system("bundle")
```
Comment (tamgrosser) [inline lib/bump.rb]: before this silently failed ... not good :D

File: spec/bump_spec.rb [hunk 2]
```diff
@@ -332,10 +332,10 @@ module Foo
       RUBY
       `git add Gemfile #{gemspec}`
       Bundler.with_clean_env { run("bundle") }
+      `git add Gemfile.lock`
```
Comment (tamgrosser) [inline spec/bump_spec.rb]: before the '--no-bundle' test passed even without the `--no-bundle` option ... making the default case bundle and all the others do something to break

**Comment by grosser** [comment]
fixes https://github.com/gregorym/bump/issues/68

@splattael @gregorym

**Comment by gregorym** [comment]
Ci is red

**Comment by grosser** [comment]
not the fault of this PR :)
rbx: `Requested binary installation but no rubies are available to download, consider skipping --binary flag.`

**Comment by splattael** [comment]
The Rubinius failure should go away with #70.

**Comment by tamgrosser** [comment]
rebased!

**Comment by splattael** [comment]
We are :green_heart: again. :+1: from me

**Pull request merged**